app/src/main/java/com/dessalines/thumbkey/utils/KeyboardLayout.kt

```diff
@@ -94,94 +94,96 @@ import com.dessalines.thumbkey.keyboards.KB_UK_THUMBKEY
+    KAThumbKey(27, KB_KA_THUMBKEY),
+    IDThumbKeySymbolsV1(28, KB_ID_THUMBKEY_SYMBOLS_V1),
+    FRMessageEase(29, KB_FR_MESSAGEEASE),
+    RUMessageEaseSymbols(30, KB_RU_MESSAGEEASE_SYMBOLS),
+    T9(31, KB_T9),
+    JAThumbKeyHiragana(32, KB_JA_THUMBKEY_HIRAGANA),
+    JAThumbKeyKatakana(33, KB_JA_THUMBKEY_KATAKANA),
+    FRThumbKeyV2(34, KB_FR_THUMBKEY_V2),
+    SVThumbKey(35, KB_SV_THUMBKEY),
+    TRThumbKey(36, KB_TR_THUMBKEY),
+    ENTypeSplit(37, KB_EN_TYPESPLIT),
+    ESTypeSplit(38, KB_ES_TYPESPLIT),
+    DETypeSplit(39, KB_DE_TYPESPLIT),
+    FRTypeSplit(40, KB_FR_TYPESPLIT),
+    ITTypeSplit(41, KB_IT_TYPESPLIT),
+    PTTypeSplit(42, KB_PT_TYPESPLIT),
+    PLTypeSplit(43, KB_PL_TYPESPLIT),
+    ENTwoHands(44, KB_EN_TWO_HANDS),
+    ENThumbKeyProgrammerWide(45, KB_EN_THUMBKEY_PROGRAMMER_WIDE),
+    HUThumbKey(46, KB_HU_THUMBKEY),
+    ESEOThumbKey(47, KB_ES_EO_THUMBKEY),
+    ITMessageEase(48, KB_IT_MESSAGEEASE),
+    ENSKThumbKey(49, KB_EN_SK_THUMBKEY),
+    HEThumbKey(50, KB_HE_THUMBKEY),
+    EOENDEThumbKey(51, KB_EOENDE_THUMBKEY),
+    GRThumbKey(52, KB_GR_THUMBKEY),
+    ENCZThumbKey(53, KB_EN_CZ_THUMBKEY),
+    ESMessageEase(54, KB_ES_MESSAGEEASE),
+    RUMessageEase(55, KB_RU_MESSAGEEASE),
+    BGThumbKeySymbols(56, KB_BG_THUMBKEY_SYMBOLS),
+    HRTwoHands(57, KB_HR_TWO_HANDS),
+    HRThumbKey(58, KB_HR_THUMBKEY),
+    HRThumbKeySymbols(59, KB_HR_THUMBKEY_SYMBOLS),
+    FITypeSplit(60, KB_FI_TYPESPLIT),
+    LVLTGThumbKey(61, KB_LV_LTG_THUMBKEY),
+    LTThumbKey(62, KB_LT_THUMBKEY),
+    IDThumbKeySymbolsV2(63, KB_ID_THUMBKEY_SYMBOLS_V2),
+    IDThumbKeySymbolsNumeric(64, KB_ID_THUMBKEY_SYMBOLS_NUMERIC_V1),
+    ESCAThumbKey(65, KB_ES_CA_THUMBKEY),
+    ENITThumbKey(66, KB_EN_IT_THUMBKEY),
+    ENEOMessageEaseSymbols(67, KB_ENEO_MESSAGEEASE_SYMBOLS),
+    UKRUMessageEaseSymbols(68, KB_UK_RU_MESSAGEEASE_SYMBOLS),
+    DEMessageEaseSymbols(69, KB_DE_MESSAGEEASE_SYMBOLS),
+    CAThumbKey(70, KB_CA_THUMBKEY),
+    MATHThumbKey(71, KB_MATH_THUMBKEY),
+    ENEEThumbKey(72, KB_EN_EE_THUMBKEY),
+    SKThumbKey(73, KB_SK_THUMBKEY),
+    ENNOTypeSplit(74, KB_EN_NO_TYPESPLIT),
+    ENThumbKeyCompose(75, KB_EN_THUMBKEY_COMPOSE),
+    ENNOThumbKey(76, KB_EN_NO_THUMBKEY),
+    ENTwoHandsSymbols(77, KB_EN_TWO_HANDS_SYMBOLS),
+    ENTwoHandsSymbolsNumbers(78, KB_EN_TWO_HANDS_SYMBOLS_NUMBERS),
+    JATypeSplitHiragana(79, KB_JA_TYPESPLIT_HIRAGANA),
+    JATypeSplitKatakana(80, KB_JA_TYPESPLIT_KATAKANA),
+    EUESThumbKey(81, KB_EU_ES_THUMBKEY),
+    ENQwertyFour(82, KB_EN_QWERTYFOUR),
+    KZThumbKey(83, KB_KZ_THUMBKEY),
+    ENMIThumbKey(84, KB_EN_MI_THUMBKEY),
+    ENMIThumbKeySymbols(85, KB_EN_MI_THUMBKEY_SYMBOLS),
+    ESCAMessageEase(86, KB_ES_CA_MESSAGEEASE),
+    SVMessageEase(87, KB_SV_MESSAGEEASE),
+    ENDoubleSymbols(88, KB_EN_DOUBLE_SYMBOLS),
+    FIMessageEase(89, KB_FI_MESSAGEEASE),
+    ENDAThumbKeySymbols(90, KB_EN_DA_THUMBKEY_SYMBOLS),
+    ENThumbKey(91, KB_BG_MESSAGEASE_PHONETIC_SYMBOLS_SHIFTED)
```

Comment:
What's going on here? why did all these get re-added and changed indexes?

Reply:
My bad, initially I thought I should add the new one first, as it's first alphabetically. Then read the guide that new layouts should be added last. So I did fix/revert that change but it seems I didn't push the commit.

Reply:
No probs.